internal/servers/controller/controller.go

```diff
@@ -55,11 +53,10 @@ type Controller struct {
 	// Used for testing and tracking worker health
 	workerStatusUpdateTimes *sync.Map
 
-	// grpc gateway server
+	// TODO: Rename.
 	gatewayServer   *grpc.Server
+	gatewayListener grpcServerListener
 	gatewayTicket   string
```

Comment:
TODO: Deal with these (Either rename to match what the objects actually are, or remove depending on final implementation)

Reply:
Addressed in [e31aa95](https://github.com/hashicorp/boundary/pull/1874/commits/e31aa9574ad44efa380d4cad5922d694ff4e8d0d)